internal/task/task/fs/mount.go

```diff
@@ -129,7 +162,7 @@ func NewMountFSTask(curvradm *cli.CurveAdm, cc *client.ClientConfig) (*task.Task
 	t.AddStep(&step.CreateContainer{
 		Image:             cc.GetContainerImage(),
 		Envs:              []string{"LD_PRELOAD=/usr/local/lib/libjemalloc.so"},
-		Command:           getMountCommand(cc, mountFSName),
+		Command:           getPreMountCommand(mountFSName) + getMountCommand(cc, mountFSName),
```

Comment:
Maybe join these command into one is better.

Reply:
> Maybe join these command into one is better.

fix